CHANGELOG.md

```diff
@@ -1,5 +1,34 @@
 # Changelog
```

Comment:
What is the intended audience of this Changelog? The DINSIC team?

Reply:
Yes, so we have some visibility on what has be done on the server lately.

Reply:
Ok 👍.